src/core/lib/iomgr/udp_server.h

```diff
@@ -58,12 +58,11 @@ typedef void (*grpc_udp_server_orphan_cb)(grpc_exec_ctx *exec_ctx,
                                           grpc_fd *emfd);
 
 /* Create a server, initially not bound to any ports */
-grpc_udp_server *grpc_udp_server_create(void);
+grpc_udp_server *grpc_udp_server_create(struct grpc_server *server);
```

Comment:
grpc_udp_server already received a grpc_server*, but previously it was passed in grpc_udp_server_start().  The change here moves it to grpc_udp_server_create() so that the grpc_server's channel_args are available to grpc_udp_server when it is creating and binding UDP sockets.  Access to the channel_args is needed since they are how the grpc_socket_factory is passed to the grpc_udp_server.

Are you objecting to the use of channel_args for passing the socket factory, or to the way in which this plumbs channel_args (earlier) through to the grpc_udp_server?  What would you suggest instead?  Thanks...